qiskit/test/mock/fake_provider.py

```diff
@@ -76,3 +76,11 @@ def __init__(self):
                           FakeAthens()]
 
         super().__init__()
+
+    def load_account(self):
+        """Fake load_account method to mirror the IBMQ provider/"""
+        pass
+
+    def providers(self):
+        """Fake providers method to mirror the IBMQ provider."""
+        return [self]
```

Comment:
These are methods of the `IBMQFactory`, not the `AccountProvider`, right?

Reply:
That's a good point, I can make a `FakeProviderFactory` class to do this and then use that to override `qiskit.IBMQ`

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4296/commits/8718e6e450977fda04fad3686ea8baf326e2953d